packages/checkbox/src/vaadin-checkbox.d.ts

```diff
@@ -45,21 +45,28 @@ export interface CheckboxEventMap extends HTMLElementEventMap, CheckboxCustomEve
  *
  * The following shadow DOM parts are available for styling:
  *
- * Part name   | Description
- * ------------|-------------
- * `checkbox`  | The element representing a stylable custom checkbox.
+ * Part name            | Description
+ * ---------------------|-------------
+ * `checkbox`           | The element representing a stylable custom checkbox
+ * `label`              | The slotted label element wrapper
+ * `helper-text`        | The slotted helper text element wrapper
```

Comment:
> Currently, theme="helper-above-field" is not implemented. 

Just FYI: The guy who requested the feature (me) does not need this :) Just in case if that question comes up whether you wanna implement it or not.